modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/button-change-status/fragment.json

```
@@ -0,0 +1,9 @@
+{
+	"configurationPath": "configuration.json",
+	"cssPath": "index.css",
+	"htmlPath": "index.html",
+	"icon": "button",
+	"jsPath": "index.js",
+	"name": "Button Change Status",
```

Please rename this to "Organization Status Update Buttons - Staff". Also rename the folder: "organization-status-update-buttons-staff"
```suggestion
	"name": "Organization Status Update Buttons - Staff",
```